.github/workflows/create-release-branch.yml

```diff
@@ -0,0 +1,46 @@
+name: Create Release Branch
+
+on:
```

Comment:
Let's add documentation on how we will trigger this. Adding it somewhere under the `contributing` folder would be good.